src/elements/Element.js

```diff
@@ -33,13 +36,24 @@ export default class Element {
         }
     }
 
+    _type: string;
+    _firstChild: ?Element;
+    _lastChild: ?Element;
+    _parentElement: ?Element | ?Program;
+    _nextSibling: ?Element;
+    _previousSibling: ?Element;
+    _childElements: Array<Element>;
+
+    value: ?string;
+    isModuleSpecifier: ?string;
+
     /**
      * Element type.
      * Important: some Node types also exist in Tokens. Do not rely on type only, use isToken, isNode.
      *
      * @returns {String}
      */
-    get type() {
+    get type(): string {
```

Comment:
There is primitive and class types - http://flowtype.org/docs/quick-reference.html#built-in-primitive-types but I think it's different